README.md

```diff
@@ -172,7 +172,7 @@ also](http://tldp.org/LDP/abs/html/declareref.html))
 
 #### Portability
 * https://github.com/duggan/shlint
-* http://manpages.ubuntu.com/manpages/natty/man1/checkbashisms.1.html
+* https://manpages.ubuntu.com/manpages/impish/en/man1/checkbashisms.1.html
```

Comment:
This seems to be an URL rewrite so the old link is still working in a browser.

Not in CLI though:
```console
❯ wget http://manpages.ubuntu.com/manpages/natty/man1/checkbashisms.1.html
--2021-09-20 10:33:01--  http://manpages.ubuntu.com/manpages/natty/man1/checkbashisms.1.html
Resolving manpages.ubuntu.com (manpages.ubuntu.com)... 91.189.88.185, 91.189.91.48, 91.189.91.49, ...
Connecting to manpages.ubuntu.com (manpages.ubuntu.com)|91.189.88.185|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-09-20 10:33:02 ERROR 404: Not Found
```